runtime/service.go

```diff
@@ -681,6 +668,33 @@ func (s *service) buildVMConfiguration(req *proto.CreateVMRequest) (*firecracker
 	return &cfg, nil
 }
 
+func (s *service) buildNonStubDrives(req *proto.CreateVMRequest) []models.Drive {
+	var builder firecracker.DrivesBuilder
+
+	if input := req.RootDrive; input != nil {
+		builder = firecracker.DrivesBuilder{}.
+			WithRootDrive(input.PathOnHost, func(d *models.Drive) {
```

Comment:
Would it be worth creating functions like `WithReadOnly(bool)`, `WithPartuuid(string)`, etc?

Reply:
Maybe yes. Would it be better to upgrade Go SDK first then?

Reply:
Opened as https://github.com/firecracker-microvm/firecracker-go-sdk/pull/140. Will update this branch after merging the change.

Reply:
Fixed on 0e858869869d6f4bf1b366adbd232d2437c60f36

Reply:
https://github.com/firecracker-microvm/firecracker-containerd/pull/283/commits/96c7417c7fe06a4d8f2c25150c83010d9e6e400c upgrades the SDK and uses the newly-added helpers.